storagedriver/s3/s3.go

```diff
@@ -195,7 +209,7 @@ func New(params DriverParameters) (*Driver, error) {
 	// 	}
 	// }
 
-	return &Driver{s3obj, bucket, params.Encrypt, params.RootDirectory}, nil
+	return &Driver{s3obj, bucket, params.ChunkSize, params.Encrypt, params.RootDirectory}, nil
```

Comment:
The `Create()` method isn't documented either despite being exported, so I know this file isn't being checked :(